docs/topics/data-analysis/data-analysis-notebooks-output-formats.md

```diff
@@ -1,35 +1,230 @@
+from the network:
+
+```none
+%use lib-ext(0.11.0-398)
+```
+
+```kotlin
+Image("https://kotlinlang.org/docs/images/kotlin-logo.png", embed = false).withWidth(300)
+```
+
+![Using external image links](external-images-output.png){width=400}
+
+### Embedded images
+
+A disadvantage of images loaded from the network is that the image disappears if the link breaks or if you lose
+the network connection. To work around that, use embedded images, for example:
+
+```kotlin
+val kotlinMascot = Image("https://blog.jetbrains.com/wp-content/uploads/2023/04/DSGN-16174-Blog-post-banner-and-promo-materials-for-post-about-Kotlin-mascot_3.png", embed = true).withWidth(400)
+kotlinMascot
+```
+
+![Using embedded images](embedded-images-output.png){width=400}
+
+## Math formulas and equations
+
+You can render mathematical formulas and equations using the LaTeX format, a typesetting system widely used in academia:
+
+1. Add the `lib-ext` library that extends the functionality of the Jupyter kernel to your notebook:
+
+   ```none
+   %use lib-ext(0.11.0-398)
+   ```
+
+2. In the new cell, run your formula: 
+    
+   ```none
+   LATEX("c^2 = a^2 + b^2 - 2 a b \\cos\\alpha")
+   ```
+
+![Using LaTeX to render mathematical formulas](latex-output.png){width=300}
+
+## Data frames
+
+With Kotlin Notebook, you can visualize structured data with data frames:
+
+1. Add the [Kotlin DataFrame](https://kotlin.github.io/dataframe/gettingstarted.html) library to your notebook:
+
+   ```none
+   %use dataframe
+   ```
+
+2. Create the data frame and run it in the new cell:
+
+    ```kotlin
+    val months = listOf(
+        "January", "February",
+        "March", "April", "May",
+        "June", "July", "August",
+        "September", "October", "November",
+        "December"
+    )
+    
+    // Sales data for different products and regions:
+    val salesLaptop = listOf(120, 130, 150, 180, 200, 220, 240, 230, 210, 190, 160, 140)
+    val salesSmartphone = listOf(90, 100, 110, 130, 150, 170, 190, 180, 160, 140, 120, 100)
+    val salesTablet = listOf(60, 70, 80, 90, 100, 110, 120, 110, 100, 90, 80, 70)
+    
+    // A data frame with columns for Month, Sales, and Product
+    val dfSales = dataFrameOf(
+        "Month" to months + months + months,
+        "Sales" to salesLaptop + salesSmartphone + salesTablet,
+        "Product" to List(12) { "Laptop" } + List(12) { "Smartphone" } + List(12) { "Tablet" },
+    )
+    ```
+
+    The data frame uses the `dataFrameOf()` function and includes the number of products (laptops, smartphones,
+    and tablets) sold in a 12-month period.
+
+3. Explore the data in your frame, for example, by finding the product and month with the highest sales:
+
+   ```none
+   dfSales.maxBy("Sales")
+   ```
+
+![Using DataFrame to visualize data](dataframe-output.png){width=500}
+
+## Charts
+
+You can create various charts directly in your Kotlin Notebook to visualize your data:
```

Comment:
Worths mentioning that the user can export Kandy plots to the file

Reply:
In Nightly it's also possible to copy plot

Reply:
And it will available after IDEA release?

Reply:
Export - yes, copy - not yet, but most likely in 2024.1.1 it will be